sys/include/net/netopt.h

```diff
@@ -391,11 +391,12 @@ typedef enum {
     NETOPT_CCA_MODE,
 
     /**
-     * @brief   (@ref netstats_t*) get statistics about sent and received packets and data of the
-     *          device or protocol
+     * @brief   (@ref netstats_t) get statistics about sent and received packets
```

Comment:
Doxygen can't find the symbol for some reason. Maybe just remove `@ref` if not easily fixable?

Reply:
I think the doc check gives the wrong line, as the error is about `@ref netstat_t` not being resolved (singular), while `@ref netstats_t` (plural) seems to be fine. Let's see if fixing the typo a few lines below fixes the issue.